WalletWasabi.Fluent/ViewModels/Settings/BitcoinTabSettingsViewModel.cs

```diff
@@ -82,6 +82,11 @@ private void ValidateCoordinatorUri(IValidationErrors errors)
 			return;
 		}
 
+		if (!coordinatorUri.EndsWith('/'))
+		{
+			coordinatorUri += "/";
+		}
```

Comment:
Manipulating the value during validation sounds fishy. 
Do you know what is the actual source of the issue? Becuase `https://api.gingerwallet.io` works perfectly fine.

![image](https://github.com/WalletWasabi/WalletWasabi/assets/16364053/d00d29f0-6563-491f-94a3-31cf237b7c21)

Reply:
Looks like it's a problem in BTCPS, I removed it